src/plugins/braintree-dropin-error-mixin.js

```diff
@@ -0,0 +1,19 @@
+export default {
+	methods: {
+		processBraintreeDropInError(trackCategory, kivaBraintreeResponse) {
+			const errorCode = kivaBraintreeResponse.errors?.[0]?.extensions?.code;
+			const errorMessage = kivaBraintreeResponse.errors?.[0]?.message;
+			const standardErrorCode = `(Braintree error: ${errorCode})`;
+			// eslint-disable-next-line max-len
+			const standardError = `There was an error processing your payment. Please try again. ${standardErrorCode}`;
+
+			// Payment method failed, unselect attempted payment method
+			this.$refs.braintreeDropInInterface.btDropinInstance.clearSelectedPaymentMethod();
```

Comment:
this seems a little out of place... but if it's global and should be executed for "any" error, I suppose it's ok.

Reply:
I agree with this actually, moved it back to the payment wrappers so the ref is at least in the same file